Documentation/IBAnimatable 4.0 Migration Guide.md

```diff
@@ -0,0 +1,126 @@
+# IBAnimatable 4.0 Migration Guide
+
+IBAnimatable 4.0 is the latest major release of IBAnimatable, a library for designing and prototyping customized UI, interaction, navigation, transition and animation for App Store ready Apps in Interface Builder. As a major release, following Semantic Versioning conventions, 4.0 introduces API-breaking changes.
+
+This guide is provided in order to ease the transition of existing applications using IBAnimatable 3.x to the latest APIs, as well as explaining the design and structure of new and updated functionality.
+
+## Requirements
+
+- iOS 8.0+
+- Xcode 8.0+
+- Swift 3.0+
+
+## Benefits of upgrading
+- **New promise-like animation APIs:** uses promise-like animation API to replace nested closures syntax.
+
+## Breaking API Changes
+IBAnimatable 4 has introduced new promise-like animation APIs. If you used the old animation APIs in code before, this document can help you identify the most changes for those APIs.
+
+### Animation APIs
+#### Animations in Interface Builder
+If you can configure the animations in Interface Builder. There is no breaking changes at all 🎉 
```

Comment:
Yes.
"If you previously configured the animations in Interface Builder"
"If your animations are configured in Interface Builder"
Either of these would be better.